src/resources/js/event-editor.js

```diff
@@ -0,0 +1,24 @@
+var tribe_events_event_editor = tribe_events_event_editor || {};
+
+/**
+ * Implements behaviours that are specific to the event editor.
+ */
+( function( obj, $ ) {
+
+	var $sticky_in_month_view_checkbox = $( 'input[name="EventShowInCalendar"]' ),
```

Comment:
As long as this file is enqueued in the footer, this should work fine. But in the event that it is moved to the header, the input fields may not exist on the page. My suggestion would be to add an `obj.init()` method and fire that off during document.ready.